CHANGELOG.unreleased.md

```diff
@@ -1081,6 +1081,37 @@ sometimes{{nogaps}}areimportant
 {{name}} is your name
 ```
 
+#### JavaScript: Break array expression in jest each function parameters([#6694] by [@sosukesuzuki])
+
+<!-- prettier-ignore -->
+```js
+// Input
+test.each([
+  { a: "1", b: 1 },
+  { a: "2", b: 2 },
+  { a: "3", b: 3 }
+])("test", ({ a, b }) => {
+  expect(Number(a)).toBe(b);
+});
+
```

Comment:
Add two more examples here please: an array of arrays and a `Map` constructor call (see https://github.com/prettier/prettier/issues/5780).